app/javascript/widget/components/PreChat/Form.vue

```diff
@@ -22,10 +22,13 @@
       :label-class="context => labelClass(context)"
       :input-class="context => inputClass(context)"
       :validation-messages="{
-        isPhoneE164OrEmpty: $t('PRE_CHAT_FORM.FIELDS.PHONE_NUMBER.VALID_ERROR'),
+        isPhoneE164OrEmpty: $t(
```

Comment:
<img width="383" alt="Screenshot 2023-06-09 at 5 41 15 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/5cae3e89-7a26-4e67-9791-344edcfe95ba">


Noticed that the validation is removed if I select country code.

To reproduce 

1/ Add phone number as a required field. 
2/ Submit the form without any input, it shows the required error.
3/ Select the country code only without the number, it wouldn't show the required error.

Reply:
@pranavrajs Fixes this with a validation of min length of 5. The value we get is with plus sign and country code from the form. Does this work?